### ISSUE TITLE
【 v1.95.0 】 cannot merge series: duplicate series found: AccountID=0, ProjectID=0, {}

### ISSUE THREAD

**Issue by jicki** [VictoriaMetrics/VictoriaMetrics#5332]
### Describe the bug

``` 
0*absent(etcd_server_leader_changes_seen_total{job=~".*etcd.*"})[15m:1m]
``` 

ERROR: 422 cannot evaluate "etcd_server_leader_changes_seen_total{job=~\".*etcd.*\"}": cannot merge series: duplicate series found: AccountID=0, ProjectID=0, {}

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/7707585/9782cd0f-90e7-4719-ae87-b55bca45eb30)

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/7707585/2e7735b7-9f40-40b2-b297-bad48f44d391)

### To Reproduce

v1.94.0 update v1.95.0

### Version

vmselect-20231115-182857-tags-v1.95.0-cluster-0-g8b01c6caf

**Comment by jicki** [comment]
v1.94.0 did not encounter this error.

**Comment by f41gh7** [comment]
Hello, thanks for reporting.

 It's related to the latest changes into SLI/SLO optimization.

 Marked release v1.95.0 as pre-release and it's not recommend to use it at production env.

**Comment by hagen1778** [comment]
@jicki as a workaround try disabling caching for range queries or rolling back vmselect version to 1.94.

**Comment by jicki** [comment]
> @jicki as a workaround try disabling caching for range queries or rolling back vmselect version to 1.94.

OKOK   roling back vmselect version to 1.94

**Comment by electrofloat** [comment]
This also broke the docker container with tag: enterprise-stable.

**Comment by valyala** [comment]
The bug should be fixed in [VictoriaMetrics v1.95.1](https://docs.victoriametrics.com/CHANGELOG.html).

> This also broke the docker image with tag: enterprise-stable.

`stable` docker images have been updated to point to v1.95.1 release.

Closing the issue as fixed.
